src/Service/MoreInfoService/AbstractMoreInfoService.php

```diff
@@ -235,71 +241,32 @@ public function moreInfo($body): MoreInfoResponse
         $boolQuery = $this->buildElasticQuery($searchParameters);
         $search = $this->index->search($boolQuery);
 
-        $this->elasticQueryTime = $search->getResponse()->getQueryTime();
+        $this->metricsService->histogram('elastica_query_time', 'Time used to run elasticsearch query', $search->getResponse()->getQueryTime(), $labels);
+
         $results = $search->getResults();
 
-        $statsStart = microtime(true);
+        $time = microtime(true);
         $this->statsLogger->info('Cover request/response', [
             'service' => 'MoreInfoService',
             'clientID' => $body->authentication->authenticationGroup,
             'remoteIP' => $this->requestStack->getCurrentRequest()->getClientIp(),
             'searchParameters' => $searchParameters,
             'fileNames' => $this->getImageUrls($results),
-            'ElasticQueryTime' => $this->elasticQueryTime,
+            'elasticQueryTime' => $this->elasticQueryTime,
         ]);
-        $this->statsTime = microtime(true) - $statsStart;
+        $this->metricsService->histogram('stats_logging_time', 'Time used logging stats', microtime(true) - $time, $labels);
```

Comment:
"Prometheus does not have any units hard coded. For better compatibility, base units should be used."
Maybe this should be 'stats_logging_duration_seconds' 

Spelling
'Time used to log stats'

Reply:
Fixed